### DISCUSSION TITLE
Helm chart apache api installation is failing

### DISCUSSION THREAD

**Discussion by hemakumar-ganumukkala** [apache/apisix#9567] [Q&A]
apisix installation is failing with below error on the kubernetes environment :

helm repo add apisix https://charts.apiseven.com && helm repo update && helm upgrade --install apisix apisix/apisix --create-namespace  --namespace apisix --set dashboard.enabled=true --set ingress-controller.enabled=true --set ingress-controller.config.apisix.serviceNamespace=apisix  --set gateway.type=NodePort
"apisix" already exists with the same configuration, skipping
Hang tight while we grab the latest from your chart repositories...
...Successfully got an update from the "apisix" chart repository
...Successfully got an update from the "bitnami" chart repository
Update Complete. ⎈Happy Helming!⎈
Release "apisix" does not exist. Installing it now.
**Error: Chart.yaml file is missing**

**Comment by Sn0rt** [comment]
can you check your working directory? if have `APISIX` directory switch to other directory
